source/all.adoc

```diff
@@ -443,7 +443,16 @@ These tests should include but are not limited to:
 
 == Deployment and Operations
 
-All environments run in the EWC.
+All environments run in the EWC. The EWC uses Virtual Machine instances with the following currently possible configuration options:
+
+* Ressources
```

Comment:
```suggestion
* Resources
```

Reply:
hm - or wasn't it ready? I now see it is still in draft mode... Maybe @myggen could take a look anyway...